### PR TITLE
Make label[for] work with selectized input

### PR DESCRIPTION
**Pull request by barretts**
this updates any existing labels pointing to the original input or select to point to the selectize input/select #658

maintained id uniqueness by appending '_selectized' string

### CODE REVIEW

**Comment by barretts** [comment]
Not sure what to do with this, took a wild guess at the tabs, running `grunt --plugins=*` and then testing locally all works. The tests in https://github.com/brianreavis/selectize.js/blob/6fe97482b0f430949f6a799ef27c68479a062d7a/test/interaction.js before and after my tests use Syn.

**Comment by dvanderfaeillie** [comment]
Any plans on merging this?

**Comment by FabianKoestring** [comment]
Any plans on merging this? Really need this feature.

**Comment by TheSharpieOne** [comment]
What is the status of this?

**Comment by Pictor13** [comment]
:+1: 

I'd like to know about that as well.
Thank you.

**Comment by joallard** [comment]
Looks like the [build](https://travis-ci.org/selectize/selectize.js/builds/56937064) is failing. Any idea what's up? Otherwise I'd be for it.

Ps. I'd prefer id prefix `-selectize` with a dash instead of underscore, no D for simplicity

**Comment by barretts** [comment]
I'm not sure it's been a while since I looked at this. I'll run it again to see.

**Comment by pzi** [comment]
@joallard  

> Ps. I'd prefer id prefix -selectize with a dash instead of underscore, no D for simplicity

Yeah, I'd prefer that, too. Keeps it consistent with the rest of the classes selectize applies.

I just cloned and tested @barretts branch locally and I had 0 failures well.

Travis says `Syn` is not defined... however it's used everywhere else in the same file?

Would be awesome if this feature could be merged in (after a little bit more than 1 year :) ) as it would improve the forms that use selectize a lot. Any chance you get to look into it @barretts? Unfortunately, I am not very familiar with testing tools and I'd have a hard time figuring out what's wrong... otherwise I'd love to help.

Thanks & cheers
Patrik

File: test/interaction.js [hunk 1]
```diff
@@ -41,6 +41,45 @@
...
+						expect(test.selectize.isFocused).to.be.equal(true);
+						done();
+					});
+			});
+
+			it('should give it focus to input', function(done) {
+				var inputId = "labeledInput";
+				$('#fixture').append('<label for="'+inputId+'">input</label>');
+				var label = $('label[for="'+inputId+'"]');
+
```
Comment (pzi) [inline test/interaction.js:69]: Minor: While I was poking through your branch, I noticed a lot of extra WS on this line :)

**Comment by joallard** [comment]
So merged.

**Pull request closed without merging**